db/seeds.rb

```
@@ -5,3 +5,30 @@
+  'https://images.pexels.com/photos/34950/pexels-photo.jpg?auto=compress&cs=tinysrgb&dpr=1&w=500',
+  'https://cdn.pixabay.com/photo/2017/04/09/09/56/avenue-2215317__340.jpg',
+  'https://media.gettyimages.com/photos/moraine-lake-in-banff-national-park-canada-picture-id500177214?s=612x612',
+  'https://upload.wikimedia.org/wikipedia/commons/thumb/4/42/Shaqi_jrvej.jpg/1200px-Shaqi_jrvej.jpg',
+  'https://images.all-free-download.com/images/graphiclarge/beautiful_nature_landscape_05_hd_picture_166223.jpg',
+  'https://cdn.pixabay.com/photo/2017/12/17/19/08/away-3024773_960_720.jpg',
+  'https://cdn.vox-cdn.com/thumbor/p-WoAsxXv2axiLdiPTDKxmA_ZvQ=/0x0:1024x668/1200x800/filters:focal(431x253:593x415)/cdn.vox-cdn.com/uploads/chorus_image/image/63654209/GettyImages_968333170.0.jpg',
+  'https://www.kalw.org/sites/kalw/files/styles/medium/public/201601/Nature-Brain.jpg',
+  'https://cdn.pixabay.com/photo/2017/01/03/19/41/forest-1950402__340.jpg',
+  'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTxY4RdkApIU3QzuIDtPnp-Jgq-l-UvM46RE5sZ2BLkbbAUY9wKNA',
+  'https://images.pexels.com/photos/371589/pexels-photo-371589.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500',
+  'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTOGHNOOZoTS1D_zWV1QF8E8KWFaeFitaPXZUFaG-DQCaEUT_EIWw',
+  'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTmHwoif96Fy_ltZYlKV1l4mwQUUrGvfI6MBPzEwljqPzmrsTZV',
+  'https://img.purch.com/w/660/aHR0cDovL3d3dy5saXZlc2NpZW5jZS5jb20vaW1hZ2VzL2kvMDAwLzA5Ny8wNzQvb3JpZ2luYWwvbW9ua2V5LWNvdW50aW5nLmpwZw==',
+  'https://aldf.org/wp-content/uploads/2018/05/baby-lamb-16x9.jpg',
+  'http://papyrus.greenville.edu/wp-content/uploads/2018/03/featyred.png',
+  'https://twistedsifter.files.wordpress.com/2018/05/horseal.jpg?w=800&h=500',
+  'https://ottawa.ca/sites/default/files/banner_petregistration2.png',
+  'https://r.hswstatic.com/w_907/gif/nonhuman-animals-jobs-1.jpg'
+]
+
+images_list.each do |url|
+  Image.create(url: url)
```

we should throw an exception if this doesn't work

create can also take an array of objects
https://apidock.com/rails/ActiveRecord/Base/create/class

---

CHanged and fixed up